### PR TITLE
Virtual Tutorial 2020 - Running calculations

### PR DESCRIPTION
**Pull request by ramirezfranciscof**
This is based on the section on running calculations of the Lausanne 2019 tutorial. I updated the `code.yml` (the vm now has quantum espresso 6.5 in a different location) and modified the text a bit to include a subsection on Troubleshooting about using the `get_builder_restart`.

Still missing is fixing the 4 references to: setting a profile in previous sections, apendix for in depth computer/code setups (appears twice) and identifiers in previous sections (pk, uuid). I don't know how you want to handle this, we can wait until these other sections are included and then I update this PR, or we can just merge this and then I can open an issue mentioning this changes are still needed for when the other sections are added.

### CODE REVIEW

**Comment by ramirezfranciscof** [comment]
TODO: Add a note mentioning that it is easy to change code/computers and link to the end section where this will be explained.

**Review by chrisjsewell — CHANGES_REQUESTED** [review]

**Comment by chrisjsewell** [comment]
Thanks @ramirezfranciscof 

> Still missing is fixing the 4 references

Yeh in the review I suggest just changing them to a "speculative" reference, which will fail for now. Then we can fix later

File: docs/pages/2020_Intro_Week/sections/running.rst [hunk 1]
```diff
@@ -4,3 +4,614 @@
 Running computations
 ********************
 
+In this section we'll be learning how to run external codes with AiiDA through calculation plugins.
+
+We will use the `Quantum Espresso <https://www.quantum-espresso.org/>`_ package to launch a simple `density functional theory <https://en.wikipedia.org/wiki/Density_functional_theory>`_ calculation of a silicon crystal using the :doi:`PBE exchange-correlation functional <10.1103/PhysRevB.54.16533>` and check its results.
+While we're going to debug these issues 'manually' here, workflows (which you'll encounter later in this tutorial) can help you avoid these issues systematically.
```
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_Intro_Week/sections/running.rst]: True that. I changed that last line into two, hopefully it is more understandable now (I didn't push this yet, I'll re-request once I do).

File: docs/pages/2020_Intro_Week/sections/running.rst [hunk 2]
```diff
@@ -4,3 +4,614 @@
...
+   The difference is that ``submit`` will hands over the calculation to the daemon running in the background, while ``run`` will execute all tasks in the current shell.
+
+   All processes in AiiDA (you will soon get to know more) can be "launched" using one of available functions:
+
+    * run
+    * run_get_node
+    * run_get_pk
+    * submit
+
+   which are explained in more detail in the `online documentation <https://aiida-core.readthedocs.io/en/latest/working/processes.html#launching-processes>`_.
```
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_Intro_Week/sections/running.rst]: Ok, I linked to the specific part of the topics that describes those 4 methods ([here](https://aiida-quantumespresso.readthedocs.io/en/stable/user_guide/calculation_plugins/pw.html)).

File: docs/pages/2020_Intro_Week/sections/running.rst [hunk 3]
```diff
@@ -4,3 +4,614 @@
...
+    $ # Anything but [0] after the Finished state signals a failure
+
+This was expected, since we used an invalid key in the input parameters.
+Situations like this happen in real life, so AiiDA provides tools to trace back to the source of the problem and correct it.
+
+A first way to proceed is to inspect the output file of PWscf.
+
+.. code:: bash
+
+    verdi calcjob outputcat <pk_number> | less
```
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_Intro_Week/sections/running.rst]: Mmm, why would you check report first? Honestly I didn't think much about this, I just left the order they appeared in the tutorial (the `verdi process report` command is mentioned a bit after that). My guess, though, is that users would be more used to interpreting the errors of the output of the program they are using, so it is not a bad idea that they check that output first, no?

File: docs/pages/2020_Intro_Week/sections/running.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_Intro_Week/sections/running.rst]: Wait, now that I see the issue you linked more clearly I'm not sure I used the right link. Where do I include the tag v1.3.0 in the following url? Instead of `/latest/` doesn't work...

https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/processes/usage.html?highlight=run_get_pk#launching-processes

[the remark this answers is not in the document]
Comment (chrisjsewell) [reply docs/pages/2020_Intro_Week/sections/running.rst]: well it won't work yet, because v1.3.0 is not released, but this works: https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/

File: docs/pages/2020_Intro_Week/sections/running.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (chrisjsewell) [reply docs/pages/2020_Intro_Week/sections/running.rst]: specifically for `PwCalculation` maybe, but more generally `verdi process report` is there to be the user facing tool for "reporting on processes" (its in the name 😜)

a) not all calculations have a default file for `outputcat`
b) it wouldn't tell you about other errors that the parser may check for, like missing files, reaching resource limits (e.g. walltime), or not meeting convergence criteria
c) in production, you'd probably be wrapping `PwCalculation`s in restart workchains, whereby you can't dorectly use `verdi calcjob outputcat`

basically `verdi process report` provides the generic report by the process/parser, then `verdi calcjob outputcat` is there if you want to dive more deeply into debugging an issue

File: docs/pages/2020_Intro_Week/sections/running.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (ramirezfranciscof) [reply docs/pages/2020_Intro_Week/sections/running.rst]: Ok, what do you prefer then? Leave the link as it is pointing to the latest doc? or leave a currently broken link to v1.3.0?

[the remark this answers is not in the document]
Comment (chrisjsewell) [reply docs/pages/2020_Intro_Week/sections/running.rst]: yep I think the broken link is best; so we don't forget

**Comment by ramirezfranciscof** [comment]
So, the only thing failing for me locally are these broken links, but I applied all changes requested.

**Review by chrisjsewell — APPROVED** [review]
